modules/src/jokes.txt

```diff
@@ -0,0 +1,24 @@
+'''
+This file contains of all jokes to be served by JARVIS
+
+Format:
+
+<question>\t\t\t<answer>\n
+             (OR)
+<entire_joke_here>\n
+
+The "\t\t\t" splits the joke as question & answer( if its Q&A type) and \n is used for new line.
```

Comment:
The 3rd point needs to be implemented depending upon how the output is shown. I do not know how would the data be shown in the final output hence I've not tweaked it.